vespalib/src/vespa/vespalib/hwaccelrated/private_helpers.hpp

```diff
@@ -24,6 +25,11 @@ populationCount(const uint64_t *a, size_t sz) {
     return count;
 }
 
+#ifdef VESPA_USE_THREAD_SANITIZER
+template<typename T, unsigned ChunkSize>
```

Comment:
A small comment about why we're disabling sanitizer instrumentation here would be nice to have for the future